konf/site.yaml

```diff
@@ -106,6 +106,9 @@ production:
         - name: SENTRY_DSN
           value: https://0293bb7fc3104e56bafd2422e155790c@sentry.is.canonical.com//13
 
+    - paths: [/security-api]
```

Comment:
Instead of `/security-api` we were planning to have `[/security/cves.json, /security/cves/*.json, /security/notices.json, /security/notices/*.json, /security/spec.json, /security/docs]`

Not sure if comma separated is the syntax to enumerate multiple paths. Also not sure if * is the way to wildcard the `cve_id` in `/security/cves/*.json` or `notice_id` for the `/security/notices/*.json`.

Reply:
Done, the wildcard should be valid.